src/stcal/jump/twopoint_difference.py

```diff
@@ -255,6 +281,29 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
                             if (gdq[integ, group, row, col + 1] & dnu_flag) == 0:
                                 gdq[integ, group, row, col + 1] =\
                                     np.bitwise_or(gdq[integ, group, row, col + 1], jump_flag)
+
+        # flag n groups after jumps above the specified thresholds to account for
+        # the transient seen after ramp jumps
+        flag_e_threshold = [after_jump_flag_e1, after_jump_flag_e2]
+        flag_groups = [after_jump_flag_n1, after_jump_flag_n2]
+
+        cr_group, cr_row, cr_col = np.where(np.bitwise_and(gdq[integ], jump_flag))
```

Comment:
by recalculating cr_row/cr_col here, you are picking up jumps that were flagged only because they were adjacent to a true jump. is this what you want to do? or should only groups after a 'true' jump be flagged?

Reply:
Yes.  All ramp jumps regardless of how they were identified need after jump flagging from the analysis I have done.  For MIRI.

Reply:
ok great, we can add another parameter later if needed to control this then